frontend/mulighetsrommet-cms/schemas/tiltaksgjennomforing.ts

```diff
@@ -115,6 +116,9 @@ export const tiltaksgjennomforing = defineType({
         "Ikke velg arrangør dersom tiltakstypen gjelder individuelle tiltak.",
       type: "reference",
       to: [{ type: "arrangor" }],
+      hidden: ({document} ) => {
+        return !isIndividueltTiltak(document.tiltakstype?._ref);
```

Comment:
Funderte på om det kunne vært mulig å utlede dette dynamisk basert på data til tiltakstypen (og f.eks. legge til et nytt felt med styredata for om det er individuell eller ikke), men det er kanskje kun `_ref` som er tilgjengelig her?

Reply:
Det er kun ref som er tilgjengelig ja. Og fikk ikke til å bruke client inne i hidden callbacket. Men samtidig føles det litt ut som det burde vært mulig.